tf_encrypted/tensor/int64.py

```diff
@@ -223,6 +224,29 @@ def squeeze(self, axis: Optional[List[int]]=None) -> 'Int64Tensor':
     def negative(self) -> 'Int64Tensor':
         return Int64Tensor(tf.negative(self.value))
 
+    def to_odd_modulus(self, factory: OddImplicitFactory) -> OddImplicitTensor:
```

Comment:
let's use casting instead, ie along the lines of https://github.com/mortendahl/tf-encrypted/blob/master/tf_encrypted/tensor/prime.py#L112-L113